miden/examples/fib/fib.masm

```diff
@@ -1,5 +1,5 @@
 begin
-    repeat.1000
+    repeat.999
         swap dup.1 add
     end
```

Comment:
I think I'd rather keep 1000 here and say we are computing 1001-st Fibonacci number in the docs. Also, maybe add a comment here to say this too?